dotnet/samples/00-Kernel-Syntax-Examples/README.md

```diff
@@ -1,37 +1,39 @@
 # Semantic Kernel syntax examples
 
 This project contains a collection of semi-random examples about various scenarios
-using SK components. 
+using SK components.
 
 The examples are ordered by number, starting with very basic examples.
 
 Most of the examples will require secrets and credentials, to access OpenAI, Azure OpenAI,
-Bing and other resources. We suggest using .NET 
+Bing and other resources. We suggest using .NET
 [Secret Manager](https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets)
 to avoid the risk of leaking secrets into the repository, branches and pull requests.
 You can also use environment variables if you prefer.
 
 To set your secrets with Secret Manager:
+
 ```
-cd dotnet/samples/KernelSyntaxExamples
+cd dotnet/samples/00-Kernel-Syntax-Examples
 
 dotnet user-secrets init
 
-dotnet user-secrets set "OpenAI:ModelId" "..."
-dotnet user-secrets set "OpenAI:ChatModelId" "..."
+dotnet user-secrets set "Global:LlmService" "..." (OpenAI, AzureOpenAI, Apim, HuggingFace, etc.)
```

Comment:
It looks like setting `Global:LlmService` is not used in `00-Kernel-Syntax-Examples`, but instead it's used in other sample projects (e.g. `02-Native-Functions`, `03-Chaining-Functions` etc.) Should we move this README file to `dotnet/samples` folder in order to union all possible settings from all samples? (Can be done in follow-up PR)

UPD: I can see that example how to add this setting already exists in each sample project README file, so maybe we can just remove it from this file, since it's not used in `00-Kernel-Syntax-Examples` project.

Also, I'm wondering if `Apim` is a valid option for this type of setting, maybe I misunderstood the purpose of it, please let me know.

Reply:
Removed! :)